test/api/capi/capi_table_functions.cpp

```diff
@@ -93,6 +98,10 @@ TEST_CASE("Test Table Functions C API", "[capi]") {
 	REQUIRE_NO_FAIL(*result);
 	REQUIRE(result->Fetch<int64_t>(0, 0) == 42);
 
+	result = tester.Query("SELECT * FROM my_function(1, my_parameter=3)");
```

Comment:
Can we add some tests for the various edge cases as well (e.g. fetching a parameter that isn't there, pushing `nullptr` into the various arguments, etc)?

Reply:
Sure thing.

Reply:
@Mytherin Is there any preference for extending the original tests vs. creating a completely new test case? For the additional tests, I would need to update the existing bind/init/function definitions. It shouldn't be a problem, but not sure if we want to just incorporate the new API functionality or leave the existing test cases as is.

Reply:
Perhaps you could create a new test case for this?

Reply:
Sounds good, thanks for the quick reply!